Tests/Storage/TestDatabase.swift

```diff
@@ -111,6 +111,45 @@ class TestDatabase: XCTestCase {
         XCTAssertEqual(db.countRows(), 0)
     }
     
+    func testRemoveOldEventsByAge() {
+        let db = createDatabase("db")
+        
+        for i in 1...5 {
+            db.insertRow(["test": i])
+        }
+        
+        Thread.sleep(forTimeInterval: 2)
+        
+        for i in 6...10 {
+            db.insertRow(["test": i])
+        }
+        
+        db.removeOldEvents(maxSize: 5, maxAge: 1)
```

Comment:
the maxSize should be greater than 5 here?

Reply:
Good catch! Have fixed it.